src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/edit.html.twig

```diff
@@ -65,6 +65,7 @@
       {{ 'This page benefits from increased performance and includes new features such as a new combination management system. Please note this is a work in progress and some features are not available yet.'|trans({}, 'Admin.Advparameters.Help') }}
     </p>
   </div>
+  <div id="contextual-notification-box"></div>
```

Comment:
Why creating a new div and not reusing #messageBoxId?

Reply:
Because an identifier can be unique on a page.

Reply:
I know that, I was wondering why you're not reusing the existing #messageBoxId div?

Reply:
Like the Edit page doesn't use the "toolbar", the messageBoxId is placed on the top of the multishop selector like there :

![image](https://user-images.githubusercontent.com/1533248/161514711-f7d2eef4-fe95-48da-b9c8-bce206ae395b.png)

Reply:
With my solution, the notification is specific displayed : 
![image](https://user-images.githubusercontent.com/1533248/161514879-3ef96f2f-f43e-46c2-9d05-6a9c84d781f6.png)

Reply:
Much clearer :)